### PR TITLE
switch to api.nodesecurity.io

### PR DESCRIPTION
**Pull request by flenter**

### CODE REVIEW

File: bin/security_audit.js [hunk 1]
```diff
@@ -1,42 +1,55 @@
+var colors = require('colors');
 var fs = require('fs');
 var request = require('request');
 var underscore = require('underscore');
 
 var shrinkwrapPath = process.argv[2];
-var shrinkwrapStream = fs.createReadStream(shrinkwrapPath);
+var shrinkwrapData = JSON.parse(fs.readFileSync(shrinkwrapPath));
```
Comment (hatchan) [inline bin/security_audit.js]: Maybe wrap in a try/catch to prevent weird errors

File: bin/security_audit.js [hunk 2]
```diff
@@ -1,42 +1,55 @@
...
-    else {
-        console.log('Vulnerable modules detected:');
-        body.forEach(function(vulnerability) {
-            console.log('Module ' + vulnerability.module + '@' + vulnerability.version + ' has a vulnerability (more information: https://nodesecurity.io/advisories/' + vulnerability.advisory.url + ')')
-        });
-        process.exit(9);
-    }
-});
+  if (body.length === 0) {
+    console.log('No vulnerable modules detected.'.green);
```
Comment (hatchan) [inline bin/security_audit.js]: is the `.green` part some extension or something?

**Comment by hatchan** [comment]
:+1:

File: bin/security_audit.js (hunk 2 above)
Comment (flenter) [inline bin/security_audit.js]: That's [colors module's](https://github.com/Marak/colors.js) magic at work. I can also import it as `colors/safe` and it won't touch the string.prototype. But then the strings need to be wrapped in function calls, which (I think) doesn't improve things.

**Pull request merged**